packages/core/src/sass/app/_common/components/account-switcher.scss

```diff
@@ -569,3 +569,10 @@
         background-color: var(--border-disabled);
     }
 }
+
+.dc-modal__container:has(.acc-switcher__wrapper) {
+    @include tablet-screen {
+        width: unset !important;
```

Comment:
We can't avoid this !important, correct?

Reply:
It doesnt override the styles without `!important`

Reply:
@farhan-nurzi-deriv please try using this className instead:
```
.dc-modal__container_accounts-switcher {
    @include tablet-screen {
        width: unset;
        right: 0;
    }
}
```

Reply:
I checked with @farhan-nurzi-deriv on this. There is an inline style from the modal. So, we have to use the `!important` to override it.